### ISSUE TITLE
RC1170 false positive on C# 5.0

### ISSUE THREAD

**Issue by RobDesideri** [JosefPihrt/Roslynator#569]
**Product and Version Used**: 
Visual Studio extension version 2.1.1

**Steps to Reproduce**:
Create a private-set property on a project targeting .NET Framework 4.0 and C# 5.0 as language version.
```csharp
public string MyProperty {get; private set; }
```

**Actual Behavior**:
Roslynator show the info: _use read-only auto property_.

**Expected Behavior**:
As C# 5.0 not support read-only auto property, the info should be hidden.
